docs/apache-airflow-providers/howto/create-custom-providers.rst

```diff
@@ -211,16 +211,15 @@ You need to do the following to turn an existing Python package into a provider
 .. exampleinclude:: /../../airflow/provider_info.schema.json
     :language: json
 
-Example ``setup.cfg``:
+Example ``pyproject.toml``:
 
-.. code-block:: cfg
+.. code-block:: toml
 
-  [options.entry_points]
-  # the function get_provider_info is defined in myproviderpackage.somemodule
-  apache_airflow_provider=
-    provider_info=myproviderpackage.somemodule:get_provider_info
+  [project.entry-points."apache_airflow_provider"]
+  provider_info = "airflow.providers.myproviderpackage.get_provider_info:get_provider_info"
```

Comment:
I believe the default Sphinx indent is three spaces (why-so-ever) - but in our RST files I see a lot of "two spaces".
```suggestion
   [project.entry-points."apache_airflow_provider"]
   provider_info = "airflow.providers.myproviderpackage.get_provider_info:get_provider_info"
```

Reply:
No problem. Updated it.